llmfoundry/callbacks/async_eval_callback.py

```diff
@@ -0,0 +1,207 @@
+# Copyright 2022 MosaicML LLM Foundry authors
+# SPDX-License-Identifier: Apache-2.0
+import logging
+import os
+from typing import Any, Dict, List, Optional, Union
+
+from composer.core import Callback, Event, State, Time
+from composer.loggers import Logger
+from composer.loggers.mosaicml_logger import (MOSAICML_PLATFORM_ENV_VAR,
+                                              RUN_NAME_ENV_VAR)
+from composer.utils import dist
+from composer.utils.misc import create_interval_scheduler
+from mcli.api.runs import \
+    ComputeConfig  # TODO: available in root in mcli 0.5.27+
```

Comment:
Should pin be higher because of this...?

Reply:
Yea will do :+1: Originally added the todo before I had made the mcli change